Basic Programs/DivisorSeries.java

```diff
@@ -1,17 +1,15 @@
-class DivisorSeries
- {
-   public static void main (String args[])
+class Test 
+{ 
+    static void printDivisors(int n) 
     { 
-      int i=2;
-      while (i<=10)
-      System.out.println(i+"/"+(i*i)+"\t");
-      i++;
-    }
-      while (i<10)
-      {
-       System.out.print(i+"/"+(i*i)+"+");
-       i++;
-      }
-       System.out.print(i+"/"+(i*i));
-     }
-  }
+        for (int i=1;i<=n;i++) 

```

Comment:
The program I created has approach for basic users, You can keep it in two forms. 
One for advanced users and mine for newbies.